akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala

```diff
@@ -69,8 +70,8 @@ abstract class MultiDcSplitBrainSpec
   val dc2 = List(third, fourth, fifth)
   var barrierCounter = 0
 
-  def splitDataCenters(notMembers: Set[RoleName]): Unit = {
-    val memberNodes = (dc1 ++ dc2).filterNot(notMembers)
+  def splitDataCenters(doNotVerify: Set[RoleName]): Unit = {
```

Comment:
Before i looked at this code i thought the nodes passed in weren't split

Reply:
Good rename